reference/shaders-msl/asm/frag/locations-components.asm.frag

```diff
@@ -11,10 +11,10 @@ struct main0_out
 struct main0_in
```

Comment:
I guess you forgot to push the original test?

Reply:
That was actually in the last patch.

Reply:
Oh ... :| you're right.